spec/features/courses/modern_languages/edit_spec.rb

```diff
@@ -150,6 +139,51 @@
     end
   end
 
+  context "when course is not a modern language course" do
+    let(:edit_modern_languages_page) do
+      PageObjects::Page::Organisations::Courses::EditModernLanguagesPage.new
+    end
+    let(:subjects) { [build(:subject, :mathematics)] }
+    let(:subjects_page) { PageObjects::Page::Organisations::CourseSubjects.new }
+    let(:modern_languages) { [japanese_subject, russian_subject, french_subject] }
+    let(:edit_options) do
+      {
+        subjects: [modern_languages_subject],
+        modern_languages: modern_languages,
+        modern_languages_subject: modern_languages_subject,
+      }
+    end
+
+    before do
+      subjects_page.load_with_course(course)
+    end
+
+    context "but is changed to a modern language course" do
+      before do
+        subjects_page.master_subject_fields.select("Modern Languages")
+        subjects_page.save.click
+      end
+
+      it "takes user through wizard and saves subjects" do
+        expect(CGI.unescape(current_url)).to eql("http://www.example.com/organisations/#{provider.provider_code}/2020/courses/#{course.course_code}/modern-languages?course[subjects_ids][]=#{modern_languages_subject.id}")
```

Comment:
Could you just check the current path and the current query? I haven't seen other examples of where we've matched the whole URL :thinking:. Though IMO this should be split into two separate assertions

Reason for this is it feels like this is making two assertions in one here:

1. We get taken to the modern-languages page
2. The `subjects_ids` are in the query